shared/AppInsightsCommon/src/Util.ts

```diff
@@ -512,8 +513,8 @@ export class Util {
      * Checks if error has no meaningful data inside. Ususally such errors are received by window.onerror when error
      * happens in a script from other domain (cross origin, CORS).
      */
-    public static isCrossOriginError(message: string, url: string, lineNumber: number, columnNumber: number, error: Error): boolean {
-        return (message === "Script error." || message === "Script error") && !error;
+    public static isCrossOriginError(message: string|Event, url: string, lineNumber: number, columnNumber: number, error: Error): boolean {
+        return !error && _isString(message) && (message === "Script error." || message === "Script error");
```

Comment:
Technically "Script error" is not *really* a CrossOrigin error... "Script error." is the common value returned for Cross Origin errors while "Script error" occurs for some browsers but mostly when an exception occurs in a "strict" script and the browser blocks the details due to security concerns... Which also makes that type of error un-actionable as you get no details to help find the issue.

Reply:
Will address the non-cross origin issue as part of a later error overhaul change when I get to it.